iota/crypto/pycurl.py

```diff
@@ -92,14 +113,17 @@ def absorb(self, trits):
       # Move on to the next hash.
       offset += HASH_LENGTH
 
-  def squeeze(self, trits):
+  def squeeze(self, trits, offset=0):
```

Comment:
Should `squeeze` also have a `length` argument?

In practice, it seems that this argument is always `HASH_LENGTH`, but the other libraries do support it.  Examples:

- https://github.com/iotaledger/iota.lib.java/blob/v0.9.6/src/main/java/jota/pow/JCurl.java#L156
- https://github.com/iotaledger/iota.lib.js/blob/v0.4.4/lib/crypto/curl/curl.js#L78

Reply:
Added, but in real case, this doesn't use without 243 (HASH_LENGTH)